zxlive/vitem.py

```diff
@@ -256,6 +301,12 @@ def mouseReleaseEvent(self, e: QGraphicsSceneMouseEvent) -> None:
             return
         if e.button() == Qt.MouseButton.LeftButton:
             if self._old_pos != self.pos():
+                if self.g.type(self.v) == VertexType.W_INPUT:
+                    # set the position of w_in to next to w_out at the same angle
+                    w_out = get_w_partner_vitem(self.g, self.graph_scene, self.v)
+                    w_in_pos = w_out.pos() + QPointF(0, 0.3 *SCALE)
```

Comment:
Since the `0.3` also shows up here (and not only in `AddNonFlexNode`) we should definitely make it a constant